src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -596,7 +596,11 @@ protected async Task CheckoutSettingsFileAsync()
         /// </summary>
         private void LaunchSettingsFile_Changed(object sender, FileSystemEventArgs e)
         {
-            _projectFaultHandler.Forget(HandleLaunchSettingsFileChangedAsync(), _project);
+            Task task = HandleLaunchSettingsFileChangedAsync();
```

Comment:
Async methods should never return null, change HandleLaunchSettingsFileChangedAsync to return Task.CompletedTask.